docs/content/users/quickstart.md

```diff
@@ -70,6 +70,55 @@ Please note that you will need to change the PHP version to 7.4 to be able to wo
     ddev launch
     ```
 
+## Contao
+
+Further information on the DDEV procedure can also be found in the [Contao documentation](https://docs.contao.org/manual/en/guides/local-installation/ddev/).
+
+=== "Composer"
+
+    ```bash
+    mkdir my-contao-site && cd my-contao-site
+    ddev config --project-type=php --docroot=public --webserver-type=apache-fpm --php-version=8.2
+    ddev composer create contao/managed-edition:5.3
```

Comment:
Is there some reason to pin the version to 5.3?

I tested the latest, and it worked fine.

Reply:
Simply because it represents the (current) LTS version.
See: https://contao.org/en/release-plan